[2024-04-17T02:30:03.972+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-04-17T02:30:04.002+0000] {taskinstance.py:2073} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: Tutorial_MALUCO.valido scheduled__2024-04-17T01:30:00+00:00 [queued]>
[2024-04-17T02:30:04.012+0000] {taskinstance.py:2073} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: Tutorial_MALUCO.valido scheduled__2024-04-17T01:30:00+00:00 [queued]>
[2024-04-17T02:30:04.013+0000] {taskinstance.py:2303} INFO - Starting attempt 1 of 1
[2024-04-17T02:30:04.032+0000] {taskinstance.py:2327} INFO - Executing <Task(BashOperator): valido> on 2024-04-17 01:30:00+00:00
[2024-04-17T02:30:04.041+0000] {warnings.py:110} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61: DeprecationWarning: This process (pid=1790) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-04-17T02:30:04.043+0000] {standard_task_runner.py:63} INFO - Started process 1798 to run task
[2024-04-17T02:30:04.043+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'Tutorial_MALUCO', 'valido', 'scheduled__2024-04-17T01:30:00+00:00', '--job-id', '67', '--raw', '--subdir', 'DAGS_FOLDER/FirstDag.py', '--cfg-path', '/tmp/tmp8fqtxbsb']
[2024-04-17T02:30:04.045+0000] {standard_task_runner.py:91} INFO - Job 67: Subtask valido
[2024-04-17T02:30:04.108+0000] {task_command.py:426} INFO - Running <TaskInstance: Tutorial_MALUCO.valido scheduled__2024-04-17T01:30:00+00:00 [running]> on host 8dbfbc3d620a
[2024-04-17T02:30:04.221+0000] {taskinstance.py:2644} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='Tutorial_MALUCO' AIRFLOW_CTX_TASK_ID='valido' AIRFLOW_CTX_EXECUTION_DATE='2024-04-17T01:30:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-04-17T01:30:00+00:00'
[2024-04-17T02:30:04.223+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-04-17T02:30:04.224+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2024-04-17T02:30:04.226+0000] {subprocess.py:75} INFO - Running command: ['/usr/bin/bash', '-c', 'echo qualidade OK']
[2024-04-17T02:30:04.253+0000] {subprocess.py:86} INFO - Output:
[2024-04-17T02:30:04.255+0000] {subprocess.py:93} INFO - qualidade OK
[2024-04-17T02:30:04.255+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2024-04-17T02:30:04.256+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-04-17T02:30:04.289+0000] {taskinstance.py:1205} INFO - Marking task as SUCCESS. dag_id=Tutorial_MALUCO, task_id=valido, execution_date=20240417T013000, start_date=20240417T023004, end_date=20240417T023004
[2024-04-17T02:30:04.338+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-04-17T02:30:04.360+0000] {taskinstance.py:3482} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-04-17T02:30:04.364+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
